Makefile

```diff
@@ -5010,6 +5010,7 @@ LIBQPS_SRC = \
     test/cpp/qps/client_async.cc \
     test/cpp/qps/client_sync.cc \
     test/cpp/qps/driver.cc \
+    test/cpp/qps/driver_helper.cc \
```

Comment:
I do not see the `test/cpp/qps/driver_helper.cc` file in this review. Perhaps forgot to do `git add` ?